data-jdbc/src/test/groovy/io/micronaut/data/jdbc/h2/jakarta_data/persistence/PersistenceEntityTests.java

```diff
@@ -0,0 +1,395 @@
+package io.micronaut.data.jdbc.h2.jakarta_data.persistence;
+
+import io.micronaut.context.annotation.Property;
+import io.micronaut.data.jdbc.h2.H2DBProperties;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.data.Order;
+import jakarta.data.Sort;
+import jakarta.data.exceptions.EntityExistsException;
+import jakarta.data.exceptions.OptimisticLockingFailureException;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static io.micronaut.data.jdbc.h2.jakarta_data.persistence.CatalogProduct.Department.GROCERY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Execute tests with a Persistence specific entity with a repository that requires read and writes (AKA not read-only)
+ */
+@Property(name = "jpa.default.properties.hibernate.show_sql", value = "true")
```

Comment:
Does this property have effect since this is jdbc module? It's just a bit confusing to me.
Also, would it be possible to have jakarta_data tests for other dialects as well?